service/src/main/java/bio/terra/landingzone/stairway/flight/LandingZoneStepsDefinitionProviderFactory.java

```diff
@@ -0,0 +1,17 @@
+package bio.terra.landingzone.stairway.flight;
+
+public class LandingZoneStepsDefinitionProviderFactory {
+  // TODO: make enum
+  public static final String CROMWELL_BASE_DEFINITION_STEPS_PROVIDER_TYPE =
+      "CromwellBaseResourcesFactory";
+  public static final String PROTECTED_DATA_DEFINITION_STEPS_PROVIDER_NAME =
```

Comment:
I want to make sure I'm understanding this -- landing zones created with `ProtectedDataStepsDefinitionProvider` will get a `definition_id` of `ProtectedDataResourcesFactory` in the db, is that right? It's important that they do because there will be downstream code in WSM that will be making policy enforcement decisions based on whether the type of the landing zone is protected (or not).

Reply:
Yes, if you are talking about landing zone database.